ci/build_wheel_cpp.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+set -euo pipefail
+
+package_name="librmm"
+package_dir="python/librmm"
+
+source rapids-configure-sccache
+source rapids-date-string
+
+version=$(rapids-generate-version)
+commit=$(git rev-parse HEAD)
+
+RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen ${RAPIDS_CUDA_VERSION})"
+
+# This is the version of the suffix with a preceding hyphen. It's used
+# everywhere except in the final wheel name.
+PACKAGE_CUDA_SUFFIX="-${RAPIDS_PY_CUDA_SUFFIX}"
```

Comment:
I guess our tooling needs a cuda suffix, but, as noted above, I don't think the wheels we are building here are cuda version specific?

Reply:
Correct. [As mentioned above, we will revisit this](https://github.com/rapidsai/rmm/pull/1529#discussion_r1588636386).